synapse/python_dependencies.py

```diff
@@ -48,7 +48,7 @@
 # [1] https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers.
 
 REQUIREMENTS = [
-    "jsonschema>=2.5.1",
+    "jsonschema>=3.0.0",
```

Comment:
could you add a comment here to record why we need 3.0 (in a similar way to the other comments below)